src/main.py

```diff
@@ -115,27 +148,26 @@ def index():
             flask.flash(
                 "La dirección de mail no está asociada a ese padrón", "danger")
         else:
+            email = create_login_mail(email, padron)
             try:
-                email_sender.send_mail(
-                    template_path="emails/sign_in.html",
-                    subject="Enlace para consultar las notas", to_addr=email,
-                    curso=COURSE, enlace=genlink(padron))
+                email_sender.send_mail(email)
             except SendmailException as exception:
                 return flask.render_template("error.html", message=str(exception))
             else:
-                return flask.render_template("email_sent.html", email=email)
+                return flask.render_template("email_sent.html", email=email.message['To'])
+
     # TODO change wip.html for index.html when is ready for PROD
-    return flask.render_template("wip.html", form=form)
+    return flask.render_template("index.html", form=form)
```

Comment:
Ojo que aun no estamos para mostrar notas. Solo cuando hayamos filtrado en el BE